src/Storages/Hive/HiveFileMetaData.h

```diff
@@ -14,16 +14,10 @@ class HiveFileMetaData : public RemoteFileMetaDataBase
         RemoteFileMetaDataBase(schema_, cluster_, remote_path_, file_size_, last_modification_timestamp_){}
     ~HiveFileMetaData() override;
 
-    String getClassName() override { return "HiveFileMetaData"; }
+    String getClassName() const override { return "HiveFileMetaData"; }
```

Comment:
Let's name this method the same as all other methods with this purpose in our code: `getName()`

Reply:
done